pyth/plugins/rtf15/reader.py

```diff
@@ -5,15 +5,23 @@
 
 This module is potentially compatible with RTF versions up to 1.9.1,
 but may not ignore all necessary control groups.
+
+RTF is 8-bit based. Therefore, all input data is represented as
+a binary string (with b prefix for literals).
+Output data is unicode.
+Python-related constants, pyth-internal constants and error messages are plain strings
+(i.e. byte strings in Python 2, Unicode strings in Python 3):
 """
+from __future__ import absolute_import
 import string, re, itertools, struct
 
 from pyth import document
 from pyth.format import PythReader
 from pyth.encodings import symbol
+import six
 
-_CONTROLCHARS = set(string.ascii_letters + string.digits + "-*")
-_DIGITS = set(string.digits + "-")
+_CONTROLCHARS = b'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-*'
```

Comment:
Because we need them to be bytes, not strings.  (This is an important change!)
All we need is the 'in' operator, so a bytestring works just as well. And I could not find a way to create the proper set that would work on Python 2 and 3 alike.